generator/src/main/resources/line-bot-sdk-ruby-generator/model.pebble

```diff
@@ -99,17 +99,18 @@ module Line
             end
           end
 
-          def self.create(args){% if model.model.vendorExtensions.get("x-children") != null and model.model.vendorExtensions.get("x-discriminator-property") != null %}
-            klass = detect_class(args[:{{ model.model.vendorExtensions.get("x-discriminator-property") }}])
-            return klass.new(**args) if klass
-            {% endif %}
-            return new(**args)
+          def self.create(args) # steep:ignore
+            {%- if model.model.vendorExtensions.get("x-children") != null and model.model.vendorExtensions.get("x-discriminator-property") != null %}
+            klass = detect_class(type: args[:{{ model.model.vendorExtensions.get("x-discriminator-property") }}])
+            return klass.new(**args) if klass # steep:ignore
+            {%- endif %}
+            return new(**args) # steep:ignore
           end
           {%- if model.model.vendorExtensions.get("x-children") != null %}
 
           private
 
-          def self.detect_class(type)
+          def self.detect_class(type:)
```

Comment:
this is intentional. `type` is defined as keyword in RBS: https://github.com/line/line-bot-sdk-ruby/blob/a69ea471cdab86dcd043d2021394a59b5457022a/sig/line/bot/v2/webhook/model/event.rbs#L36